tests/test_non_host.py

```diff
@@ -40,10 +40,32 @@ def test_custom_interpreter(
         expected -= {"setuptools", "wheel"}
     assert found == expected, out
 
-    monkeypatch.setattr(sys, "argv", [*cmd, "--graph-output", "something"])
-    with pytest.raises(SystemExit) as context:
-        main()
-    out, err = capfd.readouterr()
-    assert context.value.code == 1
-    assert not out
-    assert err == "graphviz functionality is not supported when querying non-host python\n"
+
+def test_custom_interpreter_with_local_only(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+    capfd: pytest.CaptureFixture[str],
+) -> None:
+    venv_path = str(tmp_path / "venv")
+
+    result = virtualenv.cli_run([venv_path, "--system-site-packages", "--activators", ""])
```

Comment:
When I was adding tests, I checked the [code](https://github.com/pypa/virtualenv/blob/main/src/virtualenv/run/__init__.py#L20) and [manual](https://virtualenv.pypa.io/en/latest/user_guide.html#programmatic-api) of `virtualenv.cli_run`and it looks like it has `options` and `env` args which we can pass. However, it seems adding args to cmd is much simpler, so let's directly use this way.

Reply:
I originally wanted to construct a `foo` dist in other directory then pending it to `sys.path`, but I noticed that `system-site-packages` is enough because there are at least two dists(`packaging` and `pipdeptree`) in system dists than in venv.